src/Controller/User/RegistrationController.php

```diff
@@ -34,7 +34,9 @@ public function __invoke(
             $em->flush();
             $this->addFlash('success', 'Registration successful! You are now logged in.');
 
-            return $security->login($user, 'form_login');
+            $security->login($user, 'form_login');
+
+            return $this->forward(VerificationController::class.'::sendVerification');
```

Comment:
I don't like `forward`, but this is probably a good use, though `::` is ugly

Reply:
I'm hoping we'll be able to adjust this when we extract logic into a bundle.